src/services/firebase.js

```diff
@@ -0,0 +1,15 @@
+import firebase from 'firebase'
+require('firebase/auth')
+
+/**
+ * Initialize Firebase application with parameters
+ */
+const firebaseApp = firebase.initializeApp({
+    apiKey: "AIzaSyBziUug99HMl-cwPOzWnwHnzJxH_BmG-N8",
+    authDomain: "se701-group-5-assignment-1.firebaseapp.com",
+    projectId: "se701-group-5-assignment-1",
+    storageBucket: "se701-group-5-assignment-1.appspot.com",
+    messagingSenderId: "357803904307",
+});
+
+export default firebaseApp;
```

Comment:
Im being nitpicky, but the name 'firebaseApp' doesn't seem quite right to me. Something like 'firebaseConnection` makes more sense to me, but i'll leave it up to you if you want to change it

Reply:
not a problem, will fix right now